maintenance_timesheet/models/maintenance_request.py

```diff
@@ -0,0 +1,57 @@
+# © 2019 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, fields, models
+
+
+class MaintenanceRequest(models.Model):
+    _inherit = "maintenance.request"
+
+    timesheet_ids = fields.One2many(
+        string="Timesheets",
+        comodel_name="account.analytic.line",
+        inverse_name="maintenance_request_id",
+    )
+    timesheet_total_hours = fields.Float(compute="_compute_timesheet_total_hours")
+
+    def _add_followers(self):
+        """
+        Members of maintenance team are included as followers to automatically
+        grant request visibility and timesheet permissions for this request
+        """
+        super()._add_followers()
+        for request in self:
+            partner_ids = request.maintenance_team_id.member_ids.mapped(
+                "partner_id"
+            ).ids
+            request.message_subscribe(partner_ids=partner_ids)
+
+    @api.depends("timesheet_ids.unit_amount")
+    def _compute_timesheet_total_hours(self):
+        for request in self:
+            request.timesheet_total_hours = sum(
+                request.timesheet_ids.mapped("unit_amount")
+            )
+
+    def action_view_timesheet_ids(self):
+        """
+        Access to the current timesheets for this maintenance request
+        The view will be restricted to the current request and only HR managers
+        could create timesheets for every employee
+        """
+        self.ensure_one()
+        action = (
+            self.env.ref("maintenance_timesheet.timesheet_action_from_request")
+            .with_user(self.env.ref("base.user_admin"))
```

Comment:
Why this-???

Reply:
When testing demo user was not allowed to access to actions model, maybe is a new v14 feature. I didn't find another solution to achieve the desired behavior